### DISCUSSION TITLE
Ability to customize navigation

### DISCUSSION THREAD

**Discussion by Nocturne100** [rbreu/beeref#84] [Ideas & General Discussion]
Firstly, I must say this is great app and a decent alternative to PureRef already, thanks for making it
One shortcoming is you can't customize navigation (without compiling, at least)
For example, I'd like to have panning with Alt+MMB and zooming with Alt+RMB like it is in most 3D software
And also would be good to invert the zoom (Ctrl+MMB) to have it zoom in when you move the cursor down instead of up

**Comment by rbreu** [comment]
That's definitely been on my mind, yeah.

**Reply by AnnanDistance** [reply]
Hi,

Also it would be nice to be able to have custom modifier keys for mouse buttons.

Example : Alt + MB1 is for panning but I would like to remap it to Space + MB1 but Space is not allowed right now.

Thank you.
